app/helpers/logger/logger.js

```diff
@@ -18,31 +18,35 @@ const Logger = function (debugging, externalLogger, fileSystem) {
 
   };
 
-  const error = function (detail, errorDetail) {
+  const getMessage = function (type, detail, fileUrl) {
+    return type + ": " + detail + ((fileUrl) ? " - " + fileUrl : "");
+  };
+
+  const error = function (detail, errorDetail, fileUrl, fileName) {
     let logDatetime = getLogDatetime();
-    let message = "ERROR: " + detail + " " + errorDetail;
+    let message = getMessage("ERROR", detail + " " + errorDetail, fileUrl);
```

Comment:
@rodrigopavezi The improvements in this PR are for logging to BQ, the message is just for logging to console. Do you think file name will add benefit for console?